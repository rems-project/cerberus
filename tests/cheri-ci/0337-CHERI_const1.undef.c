int main(void)
{
  const int xxx = 42;
  int *p = (int*)&xxx;
  xxx;
  *p = 100; // undefined
}