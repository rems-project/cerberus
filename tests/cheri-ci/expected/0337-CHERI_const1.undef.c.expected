Undefined {ub: "UB064_modifying_const", stderr: "", loc: "<6:3--6:11>"}
